styles.css

```diff
@@ -0,0 +1,79 @@
+/* ****Start Mobile first**** */
+
+@media (max-width: 500px) {
+  .navbar_section {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: rgb(41, 41, 106);
+    padding-right: 40px;
+    padding-left: 20px;
+  }
+
+  .nav_botton {
+    cursor: pointer;
+  }
+
+  .dropdown_buttom {
+    position: relative;
+    display: inline;
+  }
+
+  .drop_downbtn {
+    display: none;
+    position: absolute;
+    background-color: white;
+    min-width: 160px;
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+    z-index: 1;
+  }
+
+  .drop_downbtn a {
+    color: black;
+    padding: 12px 12px;
+    text-decoration: none;
+    display: block;
+  }
+
+  .drop_downbtn a:hover {
+    background-color: rgb(69, 69, 183);
+  }
+
+  .dropdown_buttom:hover .drop_downbtn {
+    display: block;
+  }
+
+  .about_me {
+    background-image: url(./image/Header\ bg@2x.png);
+    background-repeat: no-repeat;
+    background-position: cover;
+    width: 100%;
+    font-weight: 300;
+    padding: 10px;
+    color: rgb(41, 41, 106);
+    font-size: 25px;
+  }
+
+  .my-name {
+    width: 100%;
+    font-size: 45px;
+    font-weight: 900;
+    font-family: sans-serif;
+  }
+
+  .social_media {
+    list-style: none;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .social_media_list a img {
+    padding: 20px;
+  }
+
+  .connect {
+    font-size: 25px;
+    color: rgb(84, 84, 176);
+  }
+}
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), So Kindly make sure that headline section has the proper background image.

- [ ] Kindly adjust the font size and color of "Let's connect"  and adjust alignment of social icons just like figma design.

The Figma design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/204998713-c631bba6-fd52-4f0f-85c4-0a6ddd4e8cc3.PNG)

Your Project Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/204999197-e2c06d5c-4041-4c1f-b737-233c14c7251a.PNG)

![Capture](https://user-images.githubusercontent.com/93598189/204999330-01b0c1f9-d180-404c-a21e-0b404e96b896.PNG)